docker-compose.test.yaml

```diff
@@ -10,9 +10,11 @@ services:
     command: npm run test
     volumes:
       - ./app:/home/node/app
+      - ./mocks:/home/node/mocks
       - ./test:/home/node/test
       - ./test-output:/home/node/test-output
       - ./jest.config.js:/home/node/jest.config.js
       - ./package.json:/home/node/package.json
     environment:
       RURAL_PAYMENTS_AGENCY_LAND_API_URL: https://environment.data.gov.uk
+      RURAL_PAYMENTS_API_URL: http://localhost:3100/rpp/
```

Comment:
automated tests are run using this file, do you need to start the mock server as well?

Reply:
yes it does start when the tests do